dspace-uclouvain/src/main/java/org/dspace/uclouvain/submissionMetadataGenerators/consumer/LicenseConsumer.java

```diff
@@ -0,0 +1,46 @@
+package org.dspace.uclouvain.submissionMetadataGenerators.consumer;
+
+import org.dspace.content.Bitstream;
+import org.dspace.content.factory.ContentServiceFactory;
+import org.dspace.content.service.BitstreamService;
+import org.dspace.core.Constants;
+import org.dspace.core.Context;
+import org.dspace.event.Consumer;
+import org.dspace.event.Event;
+import org.dspace.services.factory.DSpaceServicesFactory;
+
+/**
+ * Consumer used to add a default license to a bitstream that does not have one.
+ * Only adds one if the metadata field is empty.
+ * 
+ * Authored-By: Michaël Pourbaix <michael.pourbaix@uclouvain.be>
+ */
+public class LicenseConsumer implements Consumer {
+
+    String LICENSE_URL = DSpaceServicesFactory.getInstance().getConfigurationService().getProperty("uclouvain.bitstream.default.license", "https://creativecommons.org/licenses/by/4.0/");
```

Comment:
UPPER_CASE variable name should be reserve for public static variable (java best practice)